src/mocks/optimizely/index.jsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/prop-types */
+
+const useDecision = (experimentKey) => (
+  // To mock optimizely for local smoke testing, set the value of "enabled" to "true" and
+  // replace "replace_me" with the desired value from OPTIMIZELY_VTR_EXPERIMENT_VARIATION_KEYS
```

Comment:
```suggestion
  // replace "replace_me" with the desired value from OPTIMIZELY_PROMPT_EXPERIMENT_VARIATION_KEYS
```
I'd recommend making this change to reflect the variable name in this repository or drop the reference all together and just refer the reader to the `src/data/optimizely.js` file.